### PR TITLE
Introduce WorkJobs

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce WorkJobs as alias of Array<Job>.

Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
From draft #14

File: src/Project/WorkJobs.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Job} from './Job';
+
+export class WorkJobs extends Array<Job> {}
```
Comment (mhs4670go) [inline src/Project/WorkJobs.ts:19]: I'm not sure but when it comes to alias, this seems be normally used.
```suggestion
export type WorkJobs = Array<Job> {};
```

Comment (seanshpark) [reply src/Project/WorkJobs.ts:19]: I too am not sure and don't know indepth about `type` in TS, I would just like to go with class extends for now...

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**